about.html

```diff
@@ -0,0 +1,234 @@
+                              <p class="img-text">Techlife Innovative Global Summit in 2021</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>  
+
+                <div class="col-md-4 me-2 text-center">
+                    <div class="img-fluid">
+                        <div class="img-container">
+                            <img src="assets/background/past-summit-2.png" alt="Techlife 2021 summit" class="img-fluid">
+
+                            <div class="content pt-3">
+                              <h2 class="img-header">2022</h2>
+                              <p class="img-text">Techlife Innovative Global Summit in 2022</p>
+                            </div>
+                        </div>
+                    </div>              
+                </div>   
+            </div>
+        </div>
+    </section>
+
+    <!-- footer  -->
+
+    <footer id="footer" class="hideInMobile">
+        <div class="container-fluid about-footer py-5">
+                <div class="col-md-8 offset-md-2">
+                    <div class="footer-p-logo">
+                        <div class="footer-logo">
+                            <img src="assets/images/logo/techlife.png" alt="mozilla" class="img-fluid logo">
+                        </div>
+                        <p class="about-footer-p footer-p">2023 Creative Commons Korea. Some Rights Reserved.</p>
+                    </div>
+                </div> 
+        </div>
+    </footer>
+
+    <!-- show in mobile -->
+
+    
+    <!-- partners  -->
+    <section id="partners" class="showInMobile">
+        <div class="container-fluid py-5 pt-3">
+            <div class="text-center">
+                <h2 class="partners">Partner</h2>
+                <div class="line"></div>
+            </div>
+
+                <div class="col-md-10 offset-md-1 pb-3 pt-5">
+                    <div class="partners-images">
+                        <div class="text-center">
+                            <img src="assets/partner/mozilla 1.png" alt="mozilla" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/googlekorea 1.png" alt="Google" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/naver 1.png" alt="Never" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/daumkakaologo 1.png" alt="daumkakaologo" class="img-fluid">
+                        </div>
+                        <div class="text-center">
+                            <img src="assets/partner/airbnb 1.png" alt="daumkakaologo" class="img-fluid">
+                        </div>
+                    </div>
+                </div> 
+        </div>
+    </section>
```

Comment:
🖼️ Good job on updating the partner images in the project! However, there are a few adjustments that still need to be made. I would recommend reducing the image size and adjusting the gap between the images to match the design more closely.
![image](https://github.com/MozamelJawad/Capstone-Project/assets/55840999/b1952d52-e689-4fff-bdee-ed80d82ceb7d)

By reducing the image size, you will ensure that they fit well within the designated space and maintain a visually pleasing layout. Additionally, adjusting the gap between the images will provide proper spacing and alignment, contributing to the overall design consistency.

Please take the time to make these adjustments to your code, ensuring that the partner images are properly sized and spaced. Keep up the great work! 👍🎉